test/acceptance/hello.js

```diff
@@ -0,0 +1,27 @@
+var assert = require('chai').assert,
```

Comment:
ES6 for consistency with the rest of the codebase (you'll probably need 'use strict' at the top due to grunt + node4)